README.rst

```diff
@@ -44,6 +44,15 @@ Using Pip
 From Source
 -----------
 
+**Note**: pip 19.0+ is required for installation.
+To check run
+
+.. code-block:: bash
+
+    pip --version
```

Comment:
I don't think that we should document how to use Pip CLI. Just point to some proper guide like https://packaging.python.org/tutorials/installing-packages/#ensure-pip-setuptools-and-wheel-are-up-to-date.

Reply:
Then the shown usage of pip for installing could be also removed, right? (just joking)
Don't get me wrong here, but installation requires a specific version (or higher) of pip to be installed, so it's good to show the user (which might not have that kind of experience with pip) how that can be ensured/checked.
Otherwise there could just be some unspecific note like `always use the latest pip/setuptools for installing`, which doesn't ensure that this really meets the required `pip 19.0+` version.

For me it comes down to the point/question, what kind of experience does the project require from a user - and then setup documentation accordingly.

Reply:
I think it's unfair to push more maintenance burden into a project with next to 0 maintainers.

Reply:
Removed that note/block - Now everybody should be happy, right?
What's left is the note about the required pip version...